lib/bcoin.js

```diff
@@ -72,16 +73,28 @@ class LedgerBcoin {
     return this;
   }
 
+  /**
+   * Get firmware version.
+   * @returns {Object}
+   */
+
+  async getFirmwareVersion() {
+    return this.ledger.getFirmwareVersion();
+  }
+
   /**
    * Get public key.
    * @async
    * @param {(Number[]|String)} - Full derivation path
+   * @param {Boolean} [parentFingerPrint = false] - Note:
+   *  This will make it run slower.
+   *  It will request public key twice from the ledger.
    * @param {apdu.addressFlags} [addressFlags=0x00]
    * @returns {bcoin.HDPublicKey}
    * @throws {LedgerError}
    */
 
-  async getPublicKey(path, addressFlags = 0) {
+  async getPublicKey(path, parentFingerPrint = false, addressFlags = 0) {
```

Comment:
Adding `parentFingerPrint` in the middle will break the interface, it should come after `addressFlags` if we want to maintain compatibility or we need to remember to add it to a changelog and bump a major version

Reply:
I thought of not breaking it at first, but we have not reached version `1.0.0` yet, so decided it's okay to break this. It's more convenient this way. addressFlags only make sense for `verification`, other flags don't matter in our use case, so less important.